web/src/pages/api/public/prompts.ts

```diff
@@ -42,27 +43,40 @@ export default async function handler(
           projectId: authCheck.scope.projectId,
           name: searchParams.name,
           version: searchParams.version ?? undefined, // if no version is given, we take the latest active prompt
-          isActive: !searchParams.version ? true : undefined, // if no prompt is active, there will be no prompt available
+          labels: !searchParams.version
+            ? {
+                has: PRODUCTION_LABEL,
+              }
+            : undefined,
         },
       });
 
       if (!prompt) throw new LangfuseNotFoundError("Prompt not found");
 
-      return res.status(200).json(prompt);
+      return res.status(200).json({
+        ...prompt,
+        isActive: prompt.labels.includes(PRODUCTION_LABEL),
+      });
     }
 
     // Handle POST requests
     if (req.method === "POST") {
-      const input = CreatePromptSchema.parse(req.body);
+      const input = LegacyCreatePromptSchema.parse(req.body);
       const prompt = await createPrompt({
         ...input,
+        labels: input.isActive
```

Comment:
This here is the new API, why do we have isActive here? I assumed this to be gone here.